crates/runtime/src/accelerated_table.rs

```diff
@@ -68,7 +68,7 @@ pub type Result<T> = std::result::Result<T, Error>;
 // An accelerated table consists of a federated table and a local accelerator.
 //
 // The accelerator must support inserts.
```

Comment:
Let's update the comment to indicate what the oneshot channel is doing.

Reply:
Added a comment